agent/packaging/ubuntu_xenial/kontena-agent/lib/systemd/system/docker.service.d/50-kontena.conf

```diff
@@ -0,0 +1,2 @@
+[Service]
+Environment='DOCKER_OPTS=--insecure-registry="10.81.0.0/19"'
```

Comment:
we discussed this with @jakolehm and decided that we'll create drop-in unit to override the `ExecStart` with proper opts